test/pdfs/xfa_issue14144.pdf.link

```diff
@@ -0,0 +1 @@
+https://www.autoriteprotectiondonnees.be/publications/formulaire-pour-une-notification-de-fuite-de-donnees.pdf
```

Comment:
Please avoid linking to arbitrary documents on the web, since they can easily become unavailable.
In this case, please use https://github.com/mozilla/pdf.js/files/7352426/FormDB2_fr.pdf instead (from https://github.com/mozilla/pdf.js/issues/14144#issue-1027258543) and remember to add a blank-line at the end (note the warning in the GitHub UI about missing line break).

Reply:
👍🏽 will do now